.isort.cfg

```
@@ -0,0 +1,7 @@
+[settings]
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+use_parentheses=True
+line_length=88
+known_third_party=click,gym,numpy,pandas,pydantic,torch,pyspark,pytest,ruamel.yaml,sparktestingbase,scipy
```

I guess we need to put all these as [known_first_party](https://github.com/timothycrosley/isort/wiki/isort-Settings) to achieve the same order as in fbcode
```suggestion
known_first_party=click,gym,numpy,pandas,pydantic,torch,pyspark,pytest,ruamel.yaml,sparktestingbase,scipy
```

---

Or, maybe just `default_section=FIRSTPARTY`.

Also, I think we need to add `lines_after_imports=2` & `reverse_relative=True`

---

I think setting default section to THIRDPARTY worked.